internal/servicename_test.go

```diff
@@ -63,8 +65,11 @@ func TestInvalidServiceNames(t *testing.T) {
 		"no--duplication",
 		"no---duplication",
 		"endswithadash-",
+		"endswithemoji😢",
```

Comment:
oh come on lol, this might be cute but it's going to mess with vim when you have to install utf-8 nonsense